content/pages/education/index.md

```diff
@@ -40,11 +40,16 @@ majorlinks:
     - url: /education/other-educational-assistance-programs/
```

Comment:
these changes were pulled from [this](https://github.com/department-of-veterans-affairs/vets-website/pull/7947) content update

Reply:
We're going to want to remove these cross link changes for now.